source/docs/structure.md

```diff
@@ -21,58 +26,52 @@ Once initialised, here's what your project folder will look like:
 
-You can configure what storage component to use (e.g IPFS) and its config details including what node to connect to, get & upload files from, etc..
+* **contracts.js**
+This file contains the configuration for Smart Contracts, including their arguments and relationships between them, such as dependencies. Here we can also specify where to deploy our Smart Contracts and how the application should attempt to connect to a node. Please see [Configuring Contracts](contracts.html) for more details.
 
-**communication.js**
+* **storage.js**
+This file lets us configure what storage component to use (e.g IPFS), including what node to connect to, to upload and retrieve data through the application. Head over to our guide on [configuring decentralized storage](storage_configuration.html) for details configuring a decentralized storage system for our application.
 
-You can configure what communication component to use (e.g Whisper) and its config details including what node to connect to.
+* **communication.js**
+Similar to the other configuration files, this file can be used to configure what communication component should be used (e.g Whisper) and what node it should connect to. Check out our guide [message configuration](messages_configuration.html) to learn more about the available configuration options.
 
-**webserver.js**
-
-Dev webserver config, specifically the host and port to listen to, and whether to automatically open a web browser when the server first starts.
+* **webserver.js**
+This file contains the configuration for the webserver that Embark starts when developing our application. Configuration options are the host, port and other useful options, such as whether a browser window should be opened on start up or not.
 
 ### test/
 
-Tests Folder. This where you typically put the code to test smart contracts. Embark includes a [testing framework](contracts_testing.html)
+Embark lets us write tests for our Smart Contracts in JavaScript as well as [Solidity](https://github.com/ethereum/remix/tree/master/remix-tests). All of our tests are located here. To learn more about testing with Embark head over to our [guide on testing](contracts_testing.html).
 
 ### dist/
 
-The build output of your dapp will be put here. You can then distribute your app with this folder
+The build output of our application will be put here. Everything inside this folder can be considered a production ready build artifact. This folder also gets uploaded to a decentralized storage using `embark upload`.
 
 ### chains.json
 
-This file is used to keep track of the deployed contracts in each chain. See chains file documentation for more information
+This file is used to keep track of the deployed contracts in each chain. This is a super useful feature as it lets Embark figure out all by itself if and when a Smart Contract needs to be (re)deployed. See chains file documentation for more information
```

Comment:
```suggestion
This file is used to keep track of the deployed contracts in each chain. This is a super useful feature as it let's Embark figure out all by itself if and when a Smart Contract needs to be (re)deployed. See chains file documentation for more information
```

Reply:
I don't think this is correct though. 

```
Let's => Let us
Lets => letting for "it" and "it" being Embark
```

Reply:
Oh damn, you're right. My bad. English is hard.